### PR TITLE
Show uptime value in proposals quality response

### PR DESCRIPTION
**Pull request by andrey-mysterium**

### CODE REVIEW

File: market/service_proposal_test.go [hunk 1]
```diff
@@ -59,7 +60,8 @@ func Test_ServiceProposal_Serialize(t *testing.T) {
       "quality": {
         "quality": 2.0,
         "latency": 5,
-        "bandwidth": 100
+        "bandwidth": 100,
+		"uptime": 20
```
Comment (n10ty) [inline market/service_proposal_test.go:64]: broken alignment?

**Comment by n10ty** [comment]
In what units is uptime measured? Seconds? Minutes? Some index?

File: market/service_proposal_test.go (hunk 1 above)
Comment (andrey-mysterium) [reply market/service_proposal_test.go:64]: yes, it look like it broke a linter. will fix now

**Comment by andrey-mysterium** [comment]
>

**Pull request closed without merging**